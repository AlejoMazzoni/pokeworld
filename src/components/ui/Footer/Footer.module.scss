.footerContainer {
  height: 10rem;
  background-color: black;
}
