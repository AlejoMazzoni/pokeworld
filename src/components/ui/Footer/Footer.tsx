import styles from "./Footer.module.scss";

const Footer = () => {
  return (
    <div className={styles.footerContainer}>
      <div></div>
    </div>
  );
};

export default Footer;
