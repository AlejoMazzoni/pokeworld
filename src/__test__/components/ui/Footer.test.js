import renderer from "react-test-renderer";
import Footer from "../../../components/ui/Footer/Footer";

describe("Footer", () => {
  it("renders everything correctly", () => {
    const component = renderer.create(<Footer />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
