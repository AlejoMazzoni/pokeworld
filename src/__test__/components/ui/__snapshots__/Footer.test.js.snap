// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders everything correctly 1`] = `
<div
  className="footerContainer"
>
  <div />
</div>
`;
